examples/nifgen_standard_function/measurement.py

```diff
@@ -185,12 +195,15 @@ def _create_nifgen_session(
             initialization_behavior=nifgen.SessionInitializationBehavior.AUTO,
         )
 
-    return nifgen.Session(session_info.resource_name, session_info.channel_list, **session_kwargs)
+    return nifgen.Session(
+        session_info.resource_name, session_info.channel_list, options=options, **session_kwargs
```

Comment:
FYI @Tharun-Sundar @dixonjoel This Session constructor takes different arguments than the rest.